week0002/mattfanz/spiral.php

```diff
@@ -0,0 +1,72 @@
+<?php
+// FUNCTION
+function checkValidInput($input)
+{
+    if (!is_numeric($input) || $input <= 0) {
+        exit;
+    }
+}
+
+// MAIN
+$handleIn = fopen("php://stdin", "r");
+$handleOut = fopen('php://stdout', 'w');
+
+fwrite($handleOut, "Input N:");
```

Comment:
Please tell the use what M & N are. You can try this:

`Input the number of rows M:`